exchange/exchange.go

```diff
@@ -30,10 +32,11 @@ type IdFetcher interface {
 }
 
 type exchange struct {
-	adapterMap map[openrtb_ext.BidderName]adaptedBidder
-	m          *pbsmetrics.Metrics
-	cache      prebid_cache_client.Client
-	cacheTime  time.Duration
+	adapterMap    map[openrtb_ext.BidderName]adaptedBidder
+	m             *pbsmetrics.Metrics
+	cache         prebid_cache_client.Client
+	cacheTime     time.Duration
+	currencyRates []byte
```

Comment:
Wouldn't it be better to store the unmarshalled `currencyRates` rather than unmarshalling it every time it is used?

Reply:
it's only being unmarshalled once when getCurrency is called.

Reply:
No, it is unmarshalled every time `convertCurrencyWithCurrentRates`, which could be multiple times within one bid request.

See my other comment on how to avoid multiple unmarshall calls.